csv-data-management/info.json

```diff
@@ -77,16 +77,16 @@
           "visible": true,
           "editable": true,
           "value": null,
-          "description": "If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single variable",
-          "tooltip":"If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single variable"
+          "description": "If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single recordSet",
```

Comment:
How about this --> If this option selected, recordset will be return in a list of 20 batches otherwise complete result will be return in one recordset",

Reply:
its 20 batches of records, we dont know what number of records will be in single batch